projects/uswds-components/src/lib/form-field/form-field.component.ts

```diff
@@ -0,0 +1,112 @@
+import { Component, Input, ChangeDetectorRef } from '@angular/core';
+import { NgControl } from '@angular/forms';
+
+@Component({
+  selector: 'uswds-form-field',

```

Comment:
Could we update this from `uswds-form-field` to `usa-form-field` for consistency?

Reply:
Updated